mm2src/derives/enum_from/src/lib.rs

```diff
@@ -84,19 +85,85 @@ pub fn enum_from_trait(input: TokenStream) -> TokenStream {
     }
 }
 
+/// `EnumFromStringify` is very useful for generating `From<T>` trait from one enum to another enum
+/// currently, this crate can only convert enum variant with only some basic inner type such as `String`, and `Enum`
+/// type just like the example below. Can not be used for tuple, struct etc for now .
+/// More support will be added.
+///
+///
+/// ### USAGE:
+/// ```rust
+/// use enum_from::EnumFromStringify;
+/// use std::fmt::{Display, Formatter};
+///
+/// // E.G, this converts from Bar, Man to FooBar::Bar(String)
+/// #[derive(Debug, EnumFromStringify, PartialEq, Eq)]
+/// pub enum FooBar {
+///     #[from_stringify("Bar", "Man")]
+///     Bar(String),
+/// }
+///
+/// #[derive(Debug, PartialEq, Eq)]
+/// pub enum Bar {
+///     Bar(String),
+/// }
+///
+/// impl Display for Bar {
+///   fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+///        match self {
+///            Self::Bar(s) => write!(f, "{}", s),
+///        }
+///    }
+/// }
+///
+/// #[derive(Debug, PartialEq, Eq)]
+/// pub enum Man {
+///     Man(String),
+/// }
+///
+/// impl Display for Man {
+///   fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+///        match self {
+///            Self::Man(s) => write!(f, "{}", s),
+///        }
+///    }
```

Comment:
How about using `u64` and/or `std::io::Error` that implements `Display` trait already?

Reply:
done